### PR TITLE
added 'is-empty' helper

### PR DESCRIPTION
**Pull request by ryanrishi**

### CODE REVIEW

**Comment by ryanrishi** [comment]
Didn't realize that this duplicates #30 .

**Comment by jmurphyau** [comment]
I have added comments to #30 

If you can satisfy the requirements I mention in those comments, I'll merge this PR

**Comment by ryanrishi** [comment]
@jmurphyau Good idea, I made some changes to the test for better arrays and string coverage. CI is failing the ember-beta build... I'll check it out in the next few days so we can get this in.

**Comment by ryanrishi** [comment]
Hmm, it's failing because `nvm install 4` failed because the checksums didn't match...

**Comment by ryanrishi** [comment]
Cool, we're in business! I satisfied your requirements mentioned in #30 .

**Comment by ryanrishi** [comment]
@jmurphyau Do you think we can get this merged?

**Comment by jmurphyau** [comment]
@ryanrishi the tests you've done are good but you need to account for the values changing

For example, if you look in the test for the `eq` helper - it shows you a good example of how to do this..

The test simple test 2, does this:

- `valueA` = `null`, valueB = `null` - `eq` should return `true`
- sets `valueA` to `undefined` - `eq` should return `false`
- sets `valueB` to `undefined` - `eq` should return `true`
- sets `valueA` and `valueB` to `yellow` - `eq` should return `true`

**Comment by jmurphyau** [comment]
What you've done doesn't account for the values changing - it changes the values, then re-renders the template completely - this won't be the case in a live app. The template is rendered once and then ember/glimmer tracks changes and updates the html

**Comment by ryanrishi** [comment]
@jmurphyau I finally found the time to come back to this—I used a `contextChild` like some of the other tests are. Let me know if there's anything else that needs to be done. Thanks!

**Comment by ryanrishi** [comment]
@jmurphyau can you please take a look?

File: tests/unit/helpers/is-empty-test.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+});
+
+test('it works with array operations', function(assert) {
+  let contextChild = Ember.Object.create({
+    someArray: null
+  });
+  this.set('contextChild', contextChild);
+
+  this.render(
+    Ember.HTMLBars.compile("{{is-empty contextChild.someArray}}")
```
Comment (aureliosaraiva) [inline tests/unit/helpers/is-empty-test.js]: Suggestion!

Do you would can change `Ember.HTMLBars.compile` to `htmlbars-inline-precompile`?

Comment (ryanjelli) [inline tests/unit/helpers/is-empty-test.js]: @aureliosaraiva just tried this, still no luck :(

Comment (ryanrishi) [inline tests/unit/helpers/is-empty-test.js]: @aureliosaraiva just tried this, still no luck :(

**Pull request closed without merging**